R/utils.R

```diff
@@ -212,7 +212,8 @@ xml_nodes_to_lint <- function(xml, source_file, message,
   if (xml2::xml_attr(xml, "line2") == line1) {
     col2 <- as.integer(xml2::xml_attr(xml, "col2")) + offset
   } else {
-    col2 <- nchar(source_file[[line_elt]][line1])
+    # as.integer to strip names
+    col2 <- as.integer(nchar(source_file[[line_elt]][line1]))
```

Comment:
Why not `unname()` to strip names? Seems clearer to me.

Reply:
never been a fan of `unname()`, for mysterious reasons. more concretely `as.integer()` is also used in the other branch so there's some consistency too.

Reply:
In the other branch, it's used to convert a character to integer.
IMO we should use `unname` if we only want to delete the names.